src/elements/content-open-with/__tests__/ExecuteForm.test.js

```diff
@@ -1,54 +1,32 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 import ExecuteForm from '../ExecuteForm';
 
-describe('elements/content-open-with/ExecuteForm', () => {
-    afterEach(() => {
-        jest.restoreAllMocks();
-    });
-
+describe('ExecuteForm', () => {
```

Comment:
this line can stay the same
```suggestion
describe('elements/content-open-with/ExecuteForm', () => {
```